### PR TITLE
Adding support for Dynamic Entities

### PR DESCRIPTION
**Pull request by salvatorecordiano**
Reference url: https://developer.amazon.com/en-US/docs/alexa/custom-skills/use-dynamic-entities-for-customized-interactions.html

### CODE REVIEW

**Comment by salvatorecordiano** [comment]
Ready for review! @maxbeckers

File: src/Response/Directives/Dialog/Entity/TypeValue.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+        $typeValue->id       = $id;
+        $typeValue->value    = $value;
+        $typeValue->synonyms = $synonyms;
+
+        return $typeValue;
+    }
+
+    /**
+     * Specify data which should be serialized to JSON.
```
Comment (maxbeckers) [inline src/Response/Directives/Dialog/Entity/TypeValue.php]: This comment can  e replaced by `{@inheritdoc}`

Comment (salvatorecordiano) [reply src/Response/Directives/Dialog/Entity/TypeValue.php]: @maxbeckers done!

**Comment by salvatorecordiano** [comment]
@maxbeckers can you tag this version, please?

**Pull request merged**